spec/index.bs

```diff
@@ -544,7 +568,7 @@ dictionary GPULimits {
     unsigned long maxDynamicStorageBuffersPerPipelineLayout = 4;
     unsigned long maxSampledTexturesPerShaderStage = 16;
     unsigned long maxSamplersPerShaderStage = 16;
-    unsigned long maxStorageBuffersPerPipelineLayout = 8;
+    unsigned long maxStorageBuffersPerShaderStage = 4;
```

Comment:
This is a functional change, and it looks good to me.

Reply:
Yes, this is more in line with Vulkan's limits that I meant to change a while back.

Reply:
Would you mind at some point making a centralized doc (e.g. design/Limits.md) that explains how we came to each of the limits? e.g. point to the Vulkan limit that this represents. I think that info is probably all in issues, but it would be easier to find.

Reply:
Sure. To clarify, something like this should be separate from the spec itself?

Reply:
I think so. It's more of an appendix since it references the other API spec/docs. And it'll be easy to fold in later if we want.